### DISCUSSION TITLE
Best way to keep/ manage a customised version of the chat widget ?

### DISCUSSION THREAD

**Discussion by Jarrodsz** [chaskiq/chaskiq#773] [Q&A]
What would be the best way to keep a custom version of the chat widget?

We need to modify some icons and styling. 
Would it be possible to "overrule" the existing methods that output the icons for example thus not having to modify the exiting codebase. 

A bit like rails monkey patching.

**Comment by michelson** [comment]
Can you please share the changes you are trying to do and what is blocking at the moment?

**Comment by Jarrodsz** [comment]
Basically just some styling its nit witting but we need it to look a little bit more like intercom it has some parts just some better aligned and lignedout.

Also really don’t like the chat icons but that’s another personal preference. 

Perhaps we can write a feature to modify the icons true the interface lateron ( upload svg )

**Reply by michelson** [reply]
yeah, send a PR. I will be glad to accept any enhancements to the chat, thanks!
